### ISSUE TITLE
norm.max() when doing batched validation

### ISSUE THREAD

**Issue by ghost** [tomdbar/eco-dqn#9]
Shouldn't this line in mpnn.py:

edge_embeddings = F.relu(self.edge_feature_NN(torch.cat([embedded_edges, norm / norm.max()],dim=-1)))

 be replaced with something like:
max_values, _ = torch.max(norm, dim=1, keepdim=True)
edge_embeddings = F.relu(self.edge_feature_NN(torch.cat([embedded_edges, norm / max_values], dim=-1))

so that the batches are each divided by the max of their part of the norm tensor?
